cds_dojson/marc21/fields/books/multipart.py

```diff
@@ -29,6 +29,58 @@
 from cds_dojson.marc21.models.books.multipart import model
 
 
+@model.over('legacy_recid', '^001')
+def recid(self, key, value):
+    """Record Identifier."""
+    return int(value)
+
+
+@model.over('isbns', '^020__')
+@out_strip
+@for_each_value
+def isbns(self, key, value):
+    """Translates isbns stored in the record."""
+    _migration = self.get('_migration', {'volumes': []})
+    _isbns = self.get('isbns', [])
+
+    val_u = clean_val('u', value, str)
+    val_a = clean_val('a', value, str)
+
+    if val_u:
+        volume_search = \
+            re.search('(.*?)\(((v\.*|vol\.*|volume\.*|part)(\d+))\)', val_u)
```

Comment:
Could there be extra spaces in the data, like having two spaces between the dot and the number `v.  1` or space between the `v` and the dot, e.g. `v . 1`? Or would it be possible that there is no dot at all or will all data be cleaned up before the migration is done?

Reply:
I didn't see those cases in the data so possible it does not happen, also this data is being currently cleaned by @agentilb .